app/test/service.spec.ts

```diff
@@ -0,0 +1,78 @@
+import oasis from '@oasislabs/client';
+
+jest.setTimeout(20000);
+
+describe('Deployments', () => {
+  it ('should deploy non confidential', async () => {
+    let multisig = await oasis.workspace.Multisig.deploy(
+      ['b8b3666d8fea887d97ab54f571b8e5020c5c8b58'], 1,
+      { header: {confidential: false} }
```

Comment:
```suggestion
      header: {confidential: false},
```

Reply:
I need to brackets to indicate the options dict?